seqr/views/apis/family_api.py

```diff
@@ -41,12 +41,44 @@ def edit_families_handler(request, project_guid):
         return create_json_response(
             {}, status=400, reason="'families' not specified")
 
+    family_guids = [f['familyGuid'] for f in modified_families if f.get('familyGuid')]
```

Comment:
Would it be cleaner by using the code below?
```
family_guids = [x for x in [f.get('familyGuid') for f in modified_families] if x is not None]
```

Reply:
no, a single list comprehension is much faster and easier to read the a nested double list comprehension